### PR TITLE
AMDGPU: Handle gfx950 global_load_lds_* instructions

### PR DESCRIPTION
**Pull request by arsenm**
Define global_load_lds_dwordx3 and global_load_dwordx4.
Oddly it seems dwordx2 was skipped.

### CODE REVIEW

**Comment by arsenm** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/llvm/llvm-project/116680?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#116681** <a href="https://app.graphite.dev/github/pr/llvm/llvm-project/116681?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#116680** <a href="https://app.graphite.dev/github/pr/llvm/llvm-project/116680?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/llvm/llvm-project/116680?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* **#116679** <a href="https://app.graphite.dev/github/pr/llvm/llvm-project/116679?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#116678** <a href="https://app.graphite.dev/github/pr/llvm/llvm-project/116678?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#116312** <a href="https://app.graphite.dev/github/pr/llvm/llvm-project/116312?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#116311** <a href="https://app.graphite.dev/github/pr/llvm/llvm-project/116311?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#116310** <a href="https://app.graphite.dev/github/pr/llvm/llvm-project/116310?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#116309** <a href="https://app.graphite.dev/github/pr/llvm/llvm-project/116309?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#116308** <a href="https://app.graphite.dev/github/pr/llvm/llvm-project/116308?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#116307** <a href="https://app.graphite.dev/github/pr/llvm/llvm-project/116307?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: llvm/lib/Target/AMDGPU/FLATInstructions.td [hunk 1]
```diff
@@ -1980,6 +1985,10 @@ defm GLOBAL_LOAD_LDS_USHORT : FLAT_Real_AllAddr_LDS <0x028, 0x12>;
 defm GLOBAL_LOAD_LDS_SSHORT : FLAT_Real_AllAddr_LDS <0x029, 0x13>;
 defm GLOBAL_LOAD_LDS_DWORD  : FLAT_Real_AllAddr_LDS <0x02a, 0x14>;
 
+defm GLOBAL_LOAD_LDS_DWORDX3 : FLAT_Real_AllAddr_LDS <0x07e, 0x07e>;
+defm GLOBAL_LOAD_LDS_DWORDX4 : FLAT_Real_AllAddr_LDS <0x07d, 0x07d>;
+
```
Comment (shiltian) [inline llvm/lib/Target/AMDGPU/FLATInstructions.td:1990]: nit: why two blank lines?

**Review by shiltian — APPROVED** [review]

File: llvm/lib/Target/AMDGPU/FLATInstructions.td (hunk 1 above)
Comment (arsenm) [reply llvm/lib/Target/AMDGPU/FLATInstructions.td:1990]: whatever was in the merge

**Comment by arsenm** [comment]
### Merge activity

* **Nov 19, 12:46 AM EST**: A user started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/llvm/llvm-project/116680).

**Pull request merged**